sources/packages/web/src/components/generic/ModalDialogBase.vue

```diff
@@ -19,12 +19,12 @@
           </h2>
         </slot>
       </v-card-title>
-      <v-divider class="mx-6 mt-1 mb-4"></v-divider>
+      <hr class="mx-6 mt-1 mb-9 horizontal-divider" />
```

Comment:
I think we have a ticket to fix this, and first, we should try to fix the `<v-divider` issue before using the HTML tag

Reply:
I tried to fix it but I think there is an issue with <v-divider> in vue.

Reply:
if the plan is to keep this code, then pls update the bug ticket on what you did and your findings and may be we can try to fix the vuetify component issue in that story

Reply:
Reverting the code as per discussion in the call. There is a pre-existing bug for the same. [ref: https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/gh/bcgov/sims/1807]